app/views/layouts/_flashes.html.erb

```diff
@@ -0,0 +1,3 @@
+<% flash.each do |name, message| %>
+  <%= content_tag(:div, message, :id => "flash_#{name}" ) %>
```

Comment:
Right now this is displaying all the messages as one <li> item, would ideally like each message to be its own <li>. I created an issue to remind us of this need.